cafebabel/templates/articles/tags/detail.html

```diff
@@ -31,7 +31,7 @@
 
 {% block body_class %}tag-page{% endblock %}
 {% block body %}
-  <div class=tag-header data-image={{ tag.image_url }}></div>
+  <div class=tag-header data-image={{ tag.image_url or '' }}></div>
```

Comment:
Can we switch to a regular `<img>` tag? Then [enhanced](https://en.wikipedia.org/wiki/Progressive_enhancement) by JavaScript?

Reply:
Why not, but I don't know how to do the parallax effect then. And I already spent too much time on this. Plus, after reading you wikipedia link I still don't really get what you mean by _Then enhanced by JavaScript_

Reply:
I get what you were talking about now. Indeed having a `style="background-image: {{ tag.image_url }}"` could be one of those rare exception that could make some sense, and then apply the `parralax()` function onto it.